dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
In any case, this is orthogonal to the main point of this PR, so maybe let's revert it here and continue the discussion on a bug?